packages/venia-concept/src/components/Footer/footer.js

```diff
@@ -17,6 +19,7 @@ class Footer extends Component {
 
     render() {
         const { classes } = this.props;
+        const year = new Date().getFullYear();
```

Comment:
Looking back over this, I just wanted to point out that a Component's `render` function gets called a lot. Since the year is unlikely to change, we could pull the `year` out into a class- / module-level member variable and re-use it.

The call to `new Date()` isn't expensive enough to make a significant impact so this won't block a 👍 .

Reply:
@supernova-at Ah, but consider the case where the user is navigating the store _at the strike of midnight on January 1_. The cached year would not update correctly until a hard refresh re-instantiated the component. Unacceptable!!!